roles/ntp/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: Install NTP deamon
+  apt: name=ntp state=present
+
+- name: Ensure NTP is listening on correct interfaces
+  lineinfile: dest=/etc/ntp.conf line="{{ item.line }}"
```

Comment:
Von dieser Schreibweise wird in der Praxis abgeraten, besser als 
```
lineinfile:
  dest; ...
  line: ...
```

Reply:
Done